src/main/java/frc/robot/util/ShuffleBoard/Field.java

```diff
@@ -42,7 +40,7 @@ public void updateModulePositions(){
 
             modulePositions[i] = new Pose2d(postion, moduleRotation);
             if (drive.getModules()[i].getState().speedMetersPerSecond<0){
-                modulePositions[i].getRotation().plus(Rotation2d.fromDegrees(180));
+                moduleRotation = moduleRotation.plus(Rotation2d.fromDegrees(180));
```

Comment:
This changes the variable, but it doesn't change the pose. This should be done before creating the Pose2d.